src/templates/profile.ts

```diff
@@ -15,3 +16,61 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
+"""
+usecase ${name} {
+  input {
+    "field title"
+    // foo is required nullable string
+    foo! string 
+    // bar links to named field 
```

Comment:
Empty space for readability
```suggestion

    // bar links to named field 
```